ush/global_cycle.sh

```diff
@@ -233,12 +231,11 @@ OCNRES=${OCNRES:-100}
 
 #  Directories.
 gfs_ver=${gfs_ver:-v15.0.0}
-BASEDIR=${BASEDIR:-${NWROOT:-/nwprod2}}
-HOMEgfs=${HOMEgfs:-$BASEDIR/gfs_ver.${gfs_ver}}
+envir=${envir:-prod}
+HOMEgfs=${HOMEgfs:-/nw${envir}/gfs_ver.${gfs_ver}}
 EXECgfs=${EXECgfs:-$HOMEgfs/exec}
-FIX_DIR=${FIX_DIR:-$HOMEgfs/fix}
-FIXam=${FIXam:-$FIX_DIR/am}
-OROFIX=${OROFIX:-$FIX_DIR/orog/${CASE}}
+FIXgfs=${FIXgfs:-$HOMEgfs/fix}
+OROFIX=${OROFIX:-$FIXgfs/orog/${CASE}}
 FIX_SFC=${FIX_SFC:-$OROFIX/sfc}
```

Comment:
I removed both `OROFIX` and `FIX_SFC` on the global-workflow side. See comment about `FIX_SFC` replacement above.

Reply:
Done at 97b765e.